### PR TITLE
Update fonts_public.proto

### PR DESCRIPTION
**Pull request by davelab6**
Proposal to add new source section... perhaps could be part of FamilyProto tho.

Also! Slightly different topic, but our [upstream repos sheet](https://docs.google.com/spredsheets/d/1ampzD9veEdrwUMkOAJkMNkftqtv1jEygiPR0wZ6eNl8/edit#gid=0) uses repo/branch/filepath-regex, rather than repo/commit, since we are currently most interested in the latest binaries committed by the foundry rather than a specific commit. That might also be worth including here, but per ["YAGNI"](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it) principle I didn't include those fields.

### CODE REVIEW

**Comment by davelab6** [comment]
@graphicore @m4rc1e @chrissimpkins PTAL

**Comment by felipesanches** [comment]
Oh! I love this! Yes!!! :-D

**Comment by felipesanches** [comment]
I should also add a FontBakery check to ensure all font projects have this field with correct info.

File: Lib/gftools/fonts_public.proto [hunk 1]
```diff
@@ -38,3 +38,7 @@ message AxisProto {
   optional float max_value = 4;
 };
 
+message SourceProto {
+  optional string repository = 1;
+  optional string commit = 2;
+}
```
Comment (rsheeter) [inline Lib/gftools/fonts_public.proto:44]: You probably want `optional SourceProto source = 11;' on FamilyProto as well

File: Lib/gftools/fonts_public.proto [hunk 2]
```diff
@@ -38,3 +38,7 @@ message AxisProto {
   optional float max_value = 4;
 };
 
+message SourceProto {
+  optional string repository = 1;
```
Comment (rsheeter) [inline Lib/gftools/fonts_public.proto:42]: repository_url if the expectation is it always will be one. If not I suggest a comment explaining what we will find here.

Comment (davelab6) [reply Lib/gftools/fonts_public.proto]: Using repository_url

**Pull request merged**